packages/manager/modules/telecom-universe-components/src/telecom/telephony/bulkAction/modal/telephony-bulk-action-modal.controller.js

```diff
@@ -101,7 +100,8 @@ export default /* @ngInject */ function (
   self.highlightUpdatedServices = function highlightUpdatedServices(services) {
     forEach(services, (service) => {
       forEach(self.billingAccounts, (billingAccount) => {
-        const findService = find(billingAccount.services, bind('serviceName', service.serviceName));
```

Comment:
another misuse of a lodash function <3

Reply:
Is the lodash migration to v4 that introduce this bug?

Previous code:
https://github.com/ovh-ux/ng-ovh-telecom-universe-components/commit/d41f5d6cf938454a9854c442b848d99919698950#diff-b379d719f556dad58b8a1008951fbe2eR85-R97

Reply:
I don't know who wrote that and why, lodash "bind" is expecting a function as first argument but here it's a string so ... it doesn't work